README.md

```diff
@@ -1 +1,184 @@
-# Hello
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
+
+**[Hello]** is a project to test Git and GitHub features and how to set up linters for HTML and CSS.
+
+## 🛠 Built With <a name="built-with"></a>
+HTML and CSS
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  HTML and CSS
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Basic project with HTML and CSS]**
+- **[Use of Linters]**
+- **[Use of Git and GitHub]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+npm package (or install latest node.js version for windows)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+
+  cd my-folder
+  git clone git@github.com:AdrianNv/Hello.git
+
+
+
+### Install
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+
+
+
```

Comment:
As the previous reviewer requested, the Usage subsection should contain ways to run the project in the browser if while the Run tests subsections should contain the commands for running the tests written in the project. 👍

Please update according to the request! :pray: